packages/elements/src/types/story.ts

```diff
@@ -18,6 +18,7 @@
  * External dependencies
  */
 import type { AudioResource } from '@googleforcreators/media';
+import type { Taxonomy } from '@googleforcreators/story-editor/src/types/taxonomyProvider';
```

Comment:
This package must not import from `story-editor`, it would be a cyclic reference.

I fear that we have to move the `Taxonomy` related types to this package for now until we find a better place.

Reply:
Fixed --- the dashboard has one as well